meshtastic/interfaces/ble/connection.py

```
@@ -448,6 +828,228 @@ def _resolve_connect_timeout(
+        if callable(legacy_find_device) and not _is_unconfigured_mock_callable(
+            legacy_find_device
+        ):
+            return cast(BLEDevice, legacy_find_device(target_address))
+
+        underscore_find_device = getattr(self.interface, "_find_device", None)
+        if callable(underscore_find_device) and not _is_unconfigured_mock_callable(
+            underscore_find_device
+        ):
+            return cast(BLEDevice, underscore_find_device(target_address))
+
+        fallback_find_device = getattr(self.interface, "findDevice")
```

Ruff (0.15.5)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)